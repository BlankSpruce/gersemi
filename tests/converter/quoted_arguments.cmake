../inputs/quoted_arguments.cmake