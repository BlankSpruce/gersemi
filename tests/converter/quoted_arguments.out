../parser/quoted_arguments.cmake